core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -904,15 +908,15 @@ public void shutdown() {
                   new Runnable() {
                     @Override
                     public void run() {
-                      subchannel.shutdown();
+                      subchannel.shutdown(SHUTDOWN_STATUS);
```

Comment:
This looks like it should have a different status, since the channel hasn't been shutdown.